lib/cloud/file-storage/gridfsStorage.js

```diff
@@ -0,0 +1,98 @@
+var q = require('q');
+var fs = require('fs');
+var mongo = require('mongodb');
+var Grid = require('gridfs-stream');
+var MongoClient = require('mongodb').MongoClient
+
+var gfsPromise;
+
+/**
+ * Init module
+ *
+ * @param storageConfiguration configuration that should be passed from client
+ *
+ * var storageConfig = {
+ *  bucket: "raincatcher-files"
+ * }
+ */
+function init(storageConfiguration) {
+  var config = storageConfiguration;
+  if (!config || !config.mongoUrl) {
+    throw Error("Missing mongoUrl parameter for GridFs storage");
+  }
+  var deferred = q.defer();
+  MongoClient.connect(config.mongoUrl, function(err, connection) {
+    if (err) {
+      console.log("Cannot connect to mongodb server. Gridfs storage will be disabled");
```

Comment:
`console.log("Cannot connect to mongodb server. Gridfs storage will be disabled". err)`